### DISCUSSION TITLE
Archr to Seurat for trajectoires analysis

### DISCUSSION THREAD

**Discussion by A-legac45** [swaruplabUCI/ArchRtoSignac#32] [Q&A]
Hello IS it possible to convert a geneexpressionmatices from multiomic experiment preprocess by Archr to Seurat ?

Thanks

**Comment by rootze** [comment]
@A-legac45 Assuming the geneexpressionmatices you mentioned is the gene score activity imputed by ArchR using chromatin activity, yes, you can convert that `gsm <- getGeneScoreMatrix(ArchRProject = proj, SeuratObject = seurat_atac)
`
